modules/round/src/main/Takebacker.scala

```diff
@@ -68,7 +68,7 @@ final private class Takebacker(
   def no(board: TakebackBoard)(pov: Pov)(using proxy: GameProxy): Fu[(Events, TakebackBoard)] =
     pov match
       case Pov(game, color) if pov.player.isProposingTakeback =>
-        messenger.volatile(game, trans.site.takebackPropositionCanceled.txt())
+        messenger.volatile(game, pov.color.name.capitalize + ' ' + trans.site.cancelsTakeback.txt())
```

Comment:
i don't think you can glue i18n phrases together like this. you'll need to work the color into the strings as a %s

Reply:
@schlawg Good point, updated to use `%s`.